testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/base/HttpClientTest.groovy

```diff
@@ -45,17 +45,21 @@ abstract class HttpClientTest<REQUEST> extends InstrumentationSpecification {
    * for example:
    *
    * @Override
-   * int sendRequest(Request request, String method, URI uri, Map<String, String headers = [:]) {*   HttpResponse response = client.execute(request)
+   * int sendRequest(Request request, String method, URI uri, Map<String, String headers = [:]) {
+   *   HttpResponse response = client.execute(request)
    *   return response.statusCode()
-   *}*
+   * }
    * If there is no synchronous API available at all, for example as in Vert.X, a CompletableFuture
    * can be used to block on a result, for example:
    *
    * @Override
```

Comment:
Can you add `<pre>` tags around these code fragments? Hopefully that'll keep the IDE from formatting them.

Reply:
👍 🤞